src/components/InviteFriends.js

```diff
@@ -22,13 +27,17 @@ const InviteFriends = (props) => {
   }
 
   return (
-    <TextField
-      id="refer-friend-input"
-      inputRef={textFieldRef}
-      onClick={highlightReferralUrl}
-      value={referralUrl}
-      label={label}
-    />
+    <>
+      <TextField
+        id="refer-friend-input"
+        inputRef={textFieldRef}
+        value={referralUrl}
+      />
+      <FileCopyIcon
+        className={classes.copyIcon}
+        onClick={highlightReferralUrl}
```

Comment:
Either make this [copy to clipboard](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard) or remove the copy icon for now, because it'll be confusing if it just highlights the text.